packages/backend/test/unit/chart.ts

```diff
@@ -54,19 +84,21 @@ describe('Chart', () => {
 				TestIntersectionChartEntity.hour, TestIntersectionChartEntity.day,
 			],
 			migrations: ['../../migration/*.js'],
-		});
+		}) as DataSource;
 
-		await db.initialize();
+		await dataSource.initialize();
 
 		const logger = new Logger('chart'); // TODO: モックにする
-		testChart = new TestChart(db, appLockService, logger);
-		testGroupedChart = new TestGroupedChart(db, appLockService, logger);
-		testUniqueChart = new TestUniqueChart(db, appLockService, logger);
-		testIntersectionChart = new TestIntersectionChart(db, appLockService, logger);
+		testChart = new TestChart(dataSource, appLockService, logger);
+		testGroupedChart = new TestGroupedChart(dataSource, appLockService, logger);
+		testUniqueChart = new TestUniqueChart(dataSource, appLockService, logger);
+		testIntersectionChart = new TestIntersectionChart(dataSource, appLockService, logger);
 
 		clock = lolex.install({
 			now: new Date(Date.UTC(2000, 0, 1, 0, 0, 0)),
 			shouldClearNativeTimers: true,
+			// https://github.com/oguimbal/pg-mem/issues/170#issuecomment-956078966
+			toFake: ['Date'],
```

Comment:
`pg-mem` doesn't seem to work in conjunction with `@sinonjs/fake-timers`, and the query never returns and the test times out. I've used the workaround mentioned in https://github.com/oguimbal/pg-mem/issues/170#issuecomment-956078966.